push-messaging-and-notifications/service-worker.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+function sendToServer() {
+  // TODO: Implement ;)
```

Comment:
It's fine to leave this unimplemented, but could you add in a comment about how this is where the `subscription.id` and `subscription.endpoint` values need to be persisted? Folks might not understand what "sendToServer" means otherwise.